### PR TITLE
Implement-config

### PR DESCRIPTION
**Pull request by Randy424**
https://github.com/stolostron/backlog/issues/18354

- Implement config for gated features consistently.
- Cleanse 2.3.z of depricated FeatureGate resource.

### CODE REVIEW

**Comment by Randy424** [comment]
Screen Shot 2022-01-13 at 12 24 24 PM
Screen Shot 2022-01-13 at 12 25 01 PM

**Comment by Randy424** [comment]
Screen Shot 2022-01-13 at 12 24 37 PM
Screen Shot 2022-01-13 at 12 25 10 PM

File: README.md [hunk 1]
```diff
@@ -107,13 +108,12 @@ If the backend responds with a `401 Unauthorized` the frontend starts an OAuth f
...
 
 ### Single node OpenShift
 
-Enabling this feature will allow the user to create a cluster that only contains a single master node.  This option is only available for providers AWS, Azure, GCP, OpenStack and VMware when the OpenShift release image is version 4.8 or higher.
-
-To enable this feature, `oc apply -f features/single-node-openshift.yaml`.  
+Enabling this feature will allow the user to create a cluster that only contains a single control plane node. This option is only available for providers AWS, Azure, GCP, OpenStack and VMware when the OpenShift release image is version 4.8 or higher.
 
 ## References
+
```
Comment (Randy424) [inline README.md:118]: These changes come from the README.md in 2.5 main branch.

**Comment by Randy424** [comment]
@KevinFCormier I've got a couple of 2.3 loaner clusters in the wings, let me know if you'd like to test this live yourself using one.

**Review by jamestalton — APPROVED** [review]
lgtm

**Review by jamestalton — APPROVED** [review]
lgtm

**Pull request merged**